src/Configuration/tips-and-tricks/index.md

```diff
@@ -32,3 +32,18 @@ See [Bloxstrap's guide to FastFlags](https://github.com/pizzaboxer/bloxstrap/wik
 - `WINEESYNC`: allows Wine Staging to use Esync, please see [HowToEsync](https://github.com/lutris/docs/blob/master/HowToEsync.md) for more information.
 - `WINEDEBUG`: for performance reasons, this has been set to `-all`, which disables most of the logging. In order to debug crashes of Wine, it is recommended to set this to an empty string (`""`).
 - `DXVK_HUD`: is a variable used by DXVK for a hud, for more information about it you can see the [DXVK README](https://github.com/doitsujin/dxvk#hud), which includes various other variables that can be set. This can be set, for example, to `fps` to display an fps counter.
+
+## Studio backup
+
+Before you delete your wineprefixes via `vinegar delete` or `flatpak run org.vinegarhq.Vinegar delete`, you should backup data you have on your Studio prefix. This includes your settings, files you have stored etc.
```

Comment:
@apprehensions Is Studio saving local files on the prefix? We might want to change that.

--

In this case, I don't think the `etc.` is needed as there's actually no other kinds of data, so I'd word it more like:
```suggestion
Before you delete your wineprefixes via `vinegar delete` or `flatpak run org.vinegarhq.Vinegar delete`, you should backup data you have on your Studio prefix. This includes your settings and files you've stored.
```